detox/ios/Detox/Invocation/Element.swift

```diff
@@ -167,6 +167,14 @@ class Element : NSObject {
 	func replaceText(_ text: String) {
 		view.dtx_replaceText(text)
 	}
+
+	func performAccessibilityAction(_ actionName: String) {
```

Comment:
@asafkorem, in my opinion, the code here is very straightforward, but if you have extra time – look and (perhaps) please put the seal of approval.